app/controllers/restaurants_controller.rb

```diff
@@ -6,6 +6,7 @@ class RestaurantsController < ApplicationController
   def index
     @restaurants = Restaurant.all
     gon.restaurants = @restaurants
+    @select_options = Restaurant::PERMITTED_CATEGORIES.dup.insert(0, 'All Restaurants')
```

Comment:
Also this should be placed in the `helpers/restaurants_helper.rb` and called as a method from the view.